tests/plugins/test_sendmail.py

```diff
@@ -563,8 +563,8 @@ def test_recepients_from_koji(self, monkeypatch, has_addit_address, to_koji_subm
                 'name': {},
             }
         }
-        if has_original_task:
-            meta_json['metadata']['labels']['original-koji-task-id'] = MOCK_KOJI_ORIGINAL_TASK_ID
+
+        # OSBS2 TBD
```

Comment:
There shouldn't be any usage of BUILD after this PR, why do we need to mock it?

Reply:
It's there to mock 'koji-task-id'. Fixed (mostly).